oracle/api/v1alpha1/instance_types.go

```diff
@@ -89,18 +89,31 @@ type InstanceSpec struct {
 	DBNetworkServiceOptions *DBNetworkServiceOptions `json:"dbNetworkServiceOptions,omitempty"`
 }
 
+type BackupReference struct {
+	// `namespace` is the namespace of the backup.
```

Comment:
Can you elaborate a bit more on what the namespace of a backup is? I was under the impression that backups, snapshot or otherwise, are namespace independent. 

Is `BackupReference` something we expect users to specify in the yaml manifest?

Reply:
ok, makes sense! Please update the doc for the Namespace field to indicate that it's the namespace where the backup was created.